src/spec/integration/subscriptions.spec.coffee

```diff
@@ -0,0 +1,71 @@
+{SphereClient} = require '../../lib/main'
+Config = require('../../config').config
+ironConfig = require('../../config').ironConfig
+Promise = require 'bluebird'
+debug = require('debug')('spec-integration:products')
+
+newSubscription =
+  key: 'testKey'
+  destination:
+    type: 'IronMQ'
+    uri: "#{ironConfig.iron_mq_url}/3/projects/#{ironConfig.iron_project_id}/queues/test-queue/webhook?oauth=#{ironConfig.iron_token}"
+  messages: [
+    resourceTypeId: 'inventory-entry'
+    types: ['InventoryEntryDeleted']
+  ]
+
+describe 'Integration Subscriptions', ->
+
+  beforeEach (done) ->
```

Comment:
I don't remove before test because I wanted to be consistent with other tests. They also remove `after` the test. I think it's just extra API call for nothing, right? Each test should start with clean environment and not clean up after previous tests.